util/src/mem.rs

```diff
@@ -1,10 +1,32 @@
+//!
+//! This module provides functions for copying data, concatenating byte arrays,
+//! and various traits and types that help manage values, including preventing
+//! drops, discarding results, and swapping values.
+
 use std::borrow::{Borrow, BorrowMut};
 use std::cmp::min;
 use std::mem::{forget, swap};
 use std::ops::{Deref, DerefMut};
 
-/// Concatenate two byte arrays
-// TODO: Zeroize result?
```

Comment:
Does this happen now somewhere?

Reply:
Sorry, I accidently removed the TODO. I will re-insert it